src/viam/app/logs/logs.py

```diff
@@ -0,0 +1,53 @@
+import sys
```

Comment:
This entire file does not need to be publicly available. The reason it's available for `MediaStream` is because people might want to write their own implementations. But that's not the case here. These classes can actually be made private and also dropped into a private file in the `app` directory.

Reply:
Should I make a separate pr to put it in app? is there anywhere specific it should go in the directory? Also, when you say make private do you just mean prepending the class names with "_" in addition to putting in app?

Reply:
oh and follow up, same thing: to make the file private should I just name it with a leading underscore?

Reply:
ohhhh I see when you said app directory I though you meant like the app repo lol okay cool got it.. thanks!